css/products.css

```diff
@@ -0,0 +1,103 @@
+@import url("../css/utility.css");
+@import url("../css/components/navbar.css");
+@import url("../css/components/footer.css");
+
+#main {
+  display: flex;
+  align-items: start;
+  justify-content: start;
+}
+
+.filters-area {
+  min-width: 30rem;
+  background-color: var(--site-secondary);
+  color: var(--white);
+  padding: 5rem 3rem;
+}
+
+.products-area {
+  flex: 1;
+}
+
+.filter-category,
+.filter-price,
+.filter-ratings,
+.filter-brands {
+  display: flex;
+  flex-direction: column;
+  margin: 4rem 0;
+}
+
+.filter-category select {
+  margin-top: 1rem;
+  width: 90%;
+  padding: 0.7rem 1rem;
+  font-family: inherit;
+  border-radius: 3rem;
+  font-size: 1.2rem;
+}
+
+.slider-container {
+  margin: 1rem 0 0;
+  padding: 0;
+  width: 100%;
+}
+
+.slider {
+  background-color: var(--site-primary-light);
+  min-width: 75%;
+}
+
+.checkboxes-wrapper,
+.brands-wrapper {
+  margin-top: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+}
+
+.checkbox-input,
+.brand-input {
+  font-size: 1.4rem;
+  margin-top: 0.5rem;
+}
+
+.checkbox-input label,
+.brand-input label {
+  margin-left: 0.5rem;
+}
```

Comment:
changes: use px unit if it is less than 16px.

Reply:
Done as suggested.